ocean_lib/assets/test/test_utils.py

```diff
@@ -49,25 +53,42 @@ def test_utilitary_functions_for_trusted_algorithms(publisher_ocean_instance):
         publisher_ocean_instance.config.metadata_cache_uri,
     )
 
-    assert new_publisher_trusted_algorithms is not None
-    assert len(new_publisher_trusted_algorithms) > len(publisher_trusted_algorithms)
+    assert (
+        new_publisher_trusted_algorithms is not None
+    ), "Added a new trusted algorithm failed. The list is empty."
+    assert len(new_publisher_trusted_algorithms) > len(publisher_trusted_algorithms), (
+        "Added a new trusted algorithm "
```

Comment:
Please reformulate this to be shorter.
I would suggest: "New trusted algorithm list should be longer than the old one."